src/scribe_data/check/check_query_identifiers.py

```diff
@@ -41,21 +41,21 @@ def is_valid_language(query_file: Path, lang_qid: str) -> bool:
 
     Parameters
     ----------
-        query_file : Path
-            The path to the SPARQL query file being validated.
+    query_file : Path
+        The path to the SPARQL query file being validated.
 
-        lang_qid : str
-            The QID of the language extracted from the SPARQL query.
+    lang_qid : str
+        The QID of the language extracted from the SPARQL query.
 
     Returns
     -------
-        bool
-            True if the language QID is valid, otherwise False.
+    bool
+        True if the language QID is valid, otherwise False.
 
-    Example
-    -------
-        > is_valid_language(Path("path/to/query.sparql"), "Q123456")
-        True
+    Examples
```

Comment:
The section for numpy docs is technically `Examples` ([reference](https://numpydoc.readthedocs.io/en/latest/format.html#examples)). Just marking out the changes! Nothing you did wrong :)